scripts/license-checker.docker.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+set -e
+
+for package in {.,packages/*}; do
+  package_name=$(basename "$package")
+  case $package_name in
+  '.')
+    sd_app_name='giphy-js-monorepo'
+    ;;
+  'util')
+    sd_app_name='giphy-js-utils'
+    ;;
+  *)
+    sd_app_name="giphy-js-${package_name}"
+    ;;
+  esac
+
+  printf "\033[0;32mChecking dependencies in %s\033[0m\n" "$sd_app_name"
+  docker run --rm \
```

Comment:
@pshoniuk this runs a docker container for every package?

Reply:
Yes